DefensiveItem.h

```diff
@@ -2,10 +2,10 @@
 
 #include "Item.h"
 #include "Character.h"
+#include "Utility.h"
```

Comment:
Are you using anything from this `Utility.h` in this header file? 
If not, then don't include it.
Don't Include what you Don't Use.
Alternatively, Only Include what you do use.
- [ ] revise